modules/core/src/hal_replacement.hpp

```diff
@@ -69,10 +69,14 @@
 /**
 Add: _dst[i] = src1[i] + src2[i]_ @n
 Sub: _dst[i] = src1[i] - src2[i]_
-@param src1_data,src1_step first source image data and step
-@param src2_data,src2_step second source image data and step
-@param dst_data,dst_step destination image data and step
-@param width,height dimensions of the images
+@param src1_data first source image data
```

Comment:
Is this kind of changes necessary? Doxygen supports this syntax and documentation looks fine with it: https://docs.opencv.org/4.x/d8/d6f/group__core__hal__interface__addsub.html

Reply:
It does not appear to be valid syntax: https://www.doxygen.nl/manual/commands.html#cmdparam

Reply:
This page says that it is allowed:
> Note that you can also document multiple parameters with a single \param command using a comma separated list. Here is an example:
> ```
> /** Sets the position.
>  *  @param x,y,z Coordinates of the position in 3D space.
>  */
> void setPosition(double x,double y,double z,double t)
> {
> }
> ```

Reply:
Ah, so it does!  Strange that it is not in the actual syntax format of: ` \param '['dir']' <parameter-name> { parameter description }`

Well, in any case, clang's -Wdocumentation does warn, I would not have changed it otherwise.  I'll file a bug with clang, but since -Wdocumentaion found so many other genuine problems, I'd vote for accommodating it, especially since it's already done.